### ISSUE TITLE
stuck in SPI2_SendByte

### ISSUE THREAD

**Issue by JackFrost67** [ChrisWonyeobPark/BNO080-STM32F4-SPI-LL-Driver#2]
I'm trying to figure out why the init function stuck when he is asking the packet header.
Any idea?

**Comment by ChrisWonyeobPark** [comment]
I'm sorry for late. Actually I'm not used to Github.
Maybe you should modify the BNO080_GPIO_SPI_Initialization() function.
Check your pin connection with the init code.
https://youtu.be/BkKuUxFzAQU
I hope this video can help you.

**Comment by JackFrost67** [comment]
No problem.
I've watched the video, but I can't understand why you double init the SPI/GPIO, first using CubeMX auto-generated functions and then using BNO080_GPIO_SPI_Initialization().
I've modify that function with my own spi setup, but without any kind of working solution.

**Comment by ChrisWonyeobPark** [comment]
Yes you are right. It is not necessary to init twice. Just BNO080_GPIO_SPI_Initialization() function call is OK.
Please make sure that you called BNO080_enableRotationVector(2500);
And then you can check data available in polling mode with BNO080_dataAvailable().
